src/sst/elements/memHierarchy/lineTypes.h

```diff
@@ -283,6 +283,9 @@ class L1CacheLine : public CacheLine {
             std::ostringstream str;
             str << "LLSC: " << (LLSCAtomic_ ? "Y" : "N");
             str << " Lock: (" << userLock_ << "," << eventsWaitingForLock_ << ")";
+            str << " Data: ";
+            for( auto i : data_ )
+                str << uint32_t(i) << " ";
```

Comment:
Is this just so that the printStatus includes data values? I'd almost rather add a second function since this one is used all over debug code which also prints data values if the debug level is high enough (but the values aren't printed via this function). It'll duplicate values in the output.

Reply:
Yeah, it's useful to be able to see what's happening. I didn't see this functionality anywhere else -- you say there is a way to print data values? I'm fairly certain that I turned the debug verbosity up to 99 when I was testing this...granted this is two years old now.

Reply:
I added it fairly recently...so it didn't exist two years ago. I don't think I've caught all the components yet but the MESI/MSI caches and the memory controller print data values. 

I do think we should merge the piece that adds printing the full state of the backing store. And the caches don't print values in printStatus nor is it possible to trigger it on finish() right now, which is a useful capability too. Aside from debug, memH has "verbose". If verbose is set to 3 the state of components get dumped on an SST fatal() call. I think we should add data values to that dump (and make it apply to the memory backing store too) and also have an option that if verbose=4, the state prints on finish(), even if debug isn't enabled. What do you think? Would that give you all the output you need?

Reply:
Removed all modifications to lineTypes